carbon-design-system/carbon
# feat(colors): update background tokens to be compatible with layers

Closes #10872 

Updates background tokens to the values specified by @aagonzales in https://github.com/carbon-design-system/carbon/discussions/10306

#### Changelog

**Changed**

- update background token values

#### Testing / Reviewing

- Ensure the emitted values are correct
- Verify the results in the `carbon-react-next` storybook and renders as expected compared to the screenshots from the above discussion.
